_posts/2021-08-31-no79-lockdown-hacks.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: Berlin Hack & Tell \#79 - Lockdown Hacks
+date: 2021-08-31
+time: '19:15'
+location: '[c-base](https://www.c-base.org)'
+meetupUrl: https://www.meetup.com/de-DE/Berlin-Hack-and-Tell/events/280321478/
+---
+
+
+* Subway trainer
+* [serenissimo vaccine bot](https://github.com/vrde/serenissimo) - **Hack of the Month**
+* Vietname by [strathausen](https://github.com/strathausen)
+* JupyterCpp - **Hack of the Month**
+* [OpenSSH: ssh_config](https://njh.eu/openssh) equivalents of -N, -s, -n and -f by [vog](https://github.com/vog)
```

Comment:
PR: https://github.com/openssh/openssh-portable/pull/231

Reply:
this is actually the line @vog suggested - so I will not change it